Gordon360/Gordon360.csproj

```diff
@@ -1,870 +1,866 @@
-        <ProductVersion>
-        </ProductVersion>
-        <SchemaVersion>2.0</SchemaVersion>
-        <ProjectGuid>{47DF568B-4E41-4398-BD88-B6BAB507334A}</ProjectGuid>
-        <ProjectTypeGuids>{349c5851-65df-11da-9384-00065b846f21};{fae04ec0-301f-11d3-bf4b-00c04f79efbc}</ProjectTypeGuids>
-        <OutputType>Library</OutputType>
-        <AppDesignerFolder>Properties</AppDesignerFolder>
-        <RootNamespace>Gordon360</RootNamespace>
-        <AssemblyName>Gordon360</AssemblyName>
-        <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-        <MvcBuildViews>false</MvcBuildViews>
-        <UseIISExpress>true</UseIISExpress>
-        <IISExpressSSLPort>49645</IISExpressSSLPort>
-        <IISExpressAnonymousAuthentication>enabled</IISExpressAnonymousAuthentication>
-        <IISExpressWindowsAuthentication>disabled</IISExpressWindowsAuthentication>
-        <IISExpressUseClassicPipelineMode>false</IISExpressUseClassicPipelineMode>
-        <UseGlobalApplicationHostFile />
-        <NuGetPackageImportStamp>
-        </NuGetPackageImportStamp>
-        <Use64BitIISExpress>false</Use64BitIISExpress>
-    </PropertyGroup>
-    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-        <DebugSymbols>true</DebugSymbols>
-        <DebugType>full</DebugType>
-        <Optimize>false</Optimize>
-        <OutputPath>bin\</OutputPath>
-        <DefineConstants>DEBUG;TRACE</DefineConstants>
-        <ErrorReport>prompt</ErrorReport>
-        <WarningLevel>4</WarningLevel>
-        <DocumentationFile>Documentation\Gordon360.xml</DocumentationFile>
-        <NoWarn>1591</NoWarn>
-        <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
-    </PropertyGroup>
-    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-        <DebugType>pdbonly</DebugType>
-        <Optimize>true</Optimize>
-        <OutputPath>bin\</OutputPath>
-        <DefineConstants>TRACE</DefineConstants>
-        <ErrorReport>prompt</ErrorReport>
-        <WarningLevel>4</WarningLevel>
-    </PropertyGroup>
-    <ItemGroup>
-        <Reference Include="EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
-            <HintPath>..\packages\EntityFramework.6.1.3\lib\net45\EntityFramework.dll</HintPath>
-            <Private>True</Private>
-        </Reference>
-        <Reference Include="EntityFramework.SqlServer, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
-            <HintPath>..\packages\EntityFramework.6.1.3\lib\net45\EntityFramework.SqlServer.dll</HintPath>
-            <Private>True</Private>
-        </Reference>
-        <Reference Include="log4net, Version=1.2.13.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
-            <HintPath>..\packages\log4net.2.0.3\lib\net40-full\log4net.dll</HintPath>
-        </Reference>
```

Comment:
This removal (of the log4net package reference) is the only real change to this .csproj file. Everything else is indentation changes from line 2 no longer wrapping to two lines, plus the change of localhost from 3417 to 2626, which is inconsequential when the API is being run non-locally.